stacks_queues/spec/stack_queue_spec.rb

```diff
@@ -1,5 +1,36 @@
-require 'spec_helper'
+require_relative '../stack_queue'
 
-describe StackQueue do
+RSpec.describe StackQueue do
```

Comment:
This guy will come in handy when you write `StackQueue`, just run via `rspec stacks_queues/spec/stack_queue_spec.rb`